src/reader/LikeFeedbackBox.js

```diff
@@ -1,22 +1,35 @@
 import { UMR_SOURCE } from "./ArticleReader";
 import * as s from "./ArticleReader.sc";
 import strings from "../i18n/definitions";
+import { useState } from "react";
 
 export default function LikeFeedbackBox({
   api,
   articleID,
   articleInfo,
   setArticleInfo,
   source,
+  setAnswerSubmitted,
 }) {
+  const [isHovered, setIsHovered] = useState('');
+
   function setLikedState(state) {
```

Comment:
if you move this function to `ArticleReader` you will have all the relevant information in that component, and you don't have to pass so much stuff as params here. And that simplifies quite a bit this component. Same also for the DifficultyFeedback box.